packages/sw-cli/src/cli/index.js

```diff
@@ -146,22 +151,30 @@ class SWCli {
     })
     .then((swName) => {
       serviceWorkerName = swName;
+      return this._saveConfigFile();
     })
-    .then(() => {
-      logHelper.log('Root Directory: ', rootDirectory);
-      logHelper.log('File Extensions to Cache: ', fileExtentionsToCache);
-      logHelper.log('File Manifest: ', fileManifestName);
-      logHelper.log('Service Worker: ', serviceWorkerName);
+    .then((sConfig) => {
+      saveConfig = sConfig;
+
+      logHelper.warn('Root Directory: ' + rootDirectory);
+      logHelper.warn('File Extensions to Cache: ' + fileExtentionsToCache);
+      logHelper.warn('File Manifest: ' + fileManifestName);
+      logHelper.warn('Service Worker: ' + serviceWorkerName);
+      logHelper.warn('Save to Config File: ' + saveConfig);
+      logHelper.warn('');
+
+      const relativePath = path.relative(process.cwd(), rootDirectory);
+
+      const globs = [
+        path.join(relativePath, '**', '*') +
```

Comment:
I'm reluctant to drop the relativePath from the glob pattern and because of that I would stick with path.join BUT I've swapped it with your `.replace(path.sep, '/')` approach in sw-precache.